sr_unix/ftok_sems.h

```diff
@@ -19,6 +22,9 @@
 #define	DECR_CNT_TRUE		TRUE
 #define	DECR_CNT_SAFE		2
 
+#define	IMMEDIATE_FALSE		FALSE
```

Comment:
Added the following comment. Hope that is okay.

```
     25 /* Below macros are used to pass values for the "immediate" parameter in ftok_sem* functions.
     26  * Using IMMEDIATE_FALSE instead of FALSE in the caller code makes it more readable.
     27  */
```

Reply:
Yes, it took awhile but it finally occurred to me I've seen FALSE and TRUE and other generalized constants in parameter lists without any clue as to what they mean. This is a simple and clever way to mitigate that somewhat.